### DISCUSSION TITLE
lore_process这个模型没有 在哪里下载

### DISCUSSION THREAD

**Discussion by CodeFishw** [RapidAI/TableStructureRec#85] [General]
raise FileNotFoundError(f"{model_path} does not exist!")
FileNotFoundError: /Users/codefish/code/PythonProjects/TableStructureRec/lineless_table_rec/models/lore_process.onnx does not exist! 请问这个lore_process在哪里下载

**Comment by Joker1212** [comment]
whl包中应该有，也可以在release中下载 https://github.com/RapidAI/TableStructureRec/releases/tag/v0.0.0
